7.3/Dockerfile

```diff
@@ -0,0 +1,110 @@
+# syntax = docker/dockerfile:experimental
+ARG SPRYKER_PHP_VERSION=7.3.3
+
+FROM php:${SPRYKER_PHP_VERSION}-fpm-alpine3.9
+
+ENV srcRoot /data
+
+RUN mkdir -p ${srcRoot}
+
+ARG PHP_RUN_DEPS="\
+    icu-libs \
+    libbz2 \
+    libxslt \
+    libpng \
+    freetype \
+    libxpm \
+    libwebp \
+    libxpm \
+    libjpeg-turbo \
+    gmp"
+
+
+ARG PHP_BUILD_DEPS="\
+    postgresql-dev \
+    libpng-dev \
+    libwebp-dev \
+    libjpeg-turbo-dev \
+    libxpm-dev \
+    freetype-dev \
+    gmp-dev \
+    icu-dev \
+    bzip2-dev \
+    autoconf \
+    g++ \
+    make"
+
+ARG PHP_EXTENSIONS="\
+    gd \
+    gmp \
+    intl \
+    pdo_pgsql \
+    pdo_mysql \
+    mysqli \
+    pgsql \
+    bcmath \
+    bz2 \
+    sockets \
+    opcache"
+
+
+ARG CFLAGS="-I/usr/src/php"
+RUN apk update \
+    && apk add --no-cache \
+    shadow \
```

Comment:
# Required for Jenkins jobs
# TODO: Move to bas_platform image
RUN apk add procps

Reply:
Added